dev/components/appRoot.js

```diff
@@ -34,15 +35,23 @@ class AppRoot extends React.Component {
                 per_page: 100
             });
 
+            const hasPagesFilter = (repo) => {
+                return repo.has_pages;
+            };
+
             //TODO verify recommended.
             const recommendedPluginsResult = await this.octokit.search.repos({
                 q: 'topic:formit-plugin-recommended'//&sort=stars&order=desc',
             });
 
-            const publicPlugins = publicPluginsResult.data.items.filter((repo) => {
-                return repo.has_pages;
+            const developersPluginsResult = await this.octokit.search.repos({
+                //q: 'topic:formit-plugin-developers'//&sort=stars&order=desc',
+                q: developersPluginsRepos.map((devRepo) => 'repo:' + devRepo).join(' ')
```

Comment:
Interesting way to do it. Requires an extra query, but simplifies the code. Not opposed to it.